paywall/src/unlock.js/PostMessageEmitters/AccountsIframeMessageEmitter.ts

```diff
@@ -88,6 +98,7 @@ export default class AccountsIframeMessageEmitter extends FancyEmitter {
     this._postMessage = postMessage
     this._addHandler = addHandler
     this.setupListeners()
+    this.sendBufferedMessages()
```

Comment:
This is more of a question around my lack of general understanding, but is `setupListeners` blocking until all listeners are "setup"?

Reply:
Yes, `setupListeners` synchronously adds event handlers.

The terminology is a bit confusing here:
- `postMessage` is how you send something to the iframe
- `addHandler` (called by `setupListeners`) is how you receive something from the iframe